Lib/test/test_dataclasses.py

```diff
@@ -2926,24 +2926,49 @@ class C:
                 x: int
 
     def test_generated_slots_value(self):
+
```

Comment:
Since I just learned that dicts are a valid value for `__slots__`, can you add a test case where a dict is used?

Reply:
for sure. Done.